spec/lib/whitehall_importer/create_edition_spec.rb

```diff
@@ -173,6 +180,7 @@
 
         expect(edition.created_at).to eq(created_at)
         expect(edition.updated_at).to eq(updated_at)
+        expect(edition.last_edited_at).to eq(whitehall_edition.dig("revision_history", -1, "created_at"))
```

Comment:
This dig is a bit cryptic. I think we're expecting it to be the same as the updated_at value that we set it in a let? might be a bit cleaner to compare to that.